packages/medusa/src/api/admin/orders/[id]/fulfillments/[fulfillment_id]/cancel/route.ts

```diff
@@ -8,9 +8,12 @@ import {
   MedusaResponse,
 } from "../../../../../../../types/routing"
 import { AdminOrderCancelFulfillmentType } from "../../../../validators"
+import { AdditionalData } from "@medusajs/types"
 
 export const POST = async (
-  req: AuthenticatedMedusaRequest<AdminOrderCancelFulfillmentType>,
+  req: AuthenticatedMedusaRequest<
```

Comment:
thought: Shouldn't the workflow receive the additional data?

Reply:
It is receiving `additional_data` as part of the `input` because the workflow accepts the `additional_data` as a top-level property alongside the rest of the input.

Reply:
Ah yes, sorry about this. I think I am used to us typically doing `run({ input: [...req.validatedBody] })` or similar. 

Discard these comments!